docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -681,13 +681,8 @@ TABLE </sql/show-create-table>`.
 
 Table properties supply or set metadata for the underlying tables. This is key
 for {doc}`/sql/create-table-as` statements. Table properties are passed to the
-connector using a {doc}`WITH </sql/create-table-as>` clause:
```

Comment:
Unrelated: It feels a bit weird to have the _Table properties_ section after _ALTER TABLE SET PROPERTIES_ section which references some of the properties even though they haven been introduced.

Reply:
Agreed.. but not going to fix that problem here..